pages/docs/how-to-guides/links.mdx

```diff
@@ -33,7 +33,7 @@ This is an example of a single link object:
 | icon  | string | true     | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons.                                                                                     |
 | group | string | false    | Optional property to group links together, any without a group will be added to the end in a new group called "Others". If no groups are used then no groups are displayed including "Others". |
 
-To add more links add another object inside the links collection. For example:
+To add more links, add another object inside the links collection. For example:
```

Comment:
I have added the line break so `For example` is on a new line, to match the project's documentation
```suggestion
To add more links add another object inside the links collection. 

For example:
```